### PR TITLE
4997 Notification prefs wording fix

### LINKED ISSUE #4997: Notification prefs wording fix
## What went wrong? 😲

Screenshot 2024-10-04 at 12 54 54 PM

## Expected behaviour 🤔

Should be: "less than 3, or more than 6 months of stock"

## How to Reproduce 🔨

Steps to reproduce the behaviour:

1. Go to Reports
2. Click the More button

## Your environment 🌱

- Open mSupply Version: v2.3.0 RC5

### PR DESCRIPTION
**Pull request by roxy-dao**
Fixes #4997

# 👩🏻‍💻 What does this PR do?
Tweek report pref description

# 📃 Documentation

- [ ] **Part of an epic**: documentation will be completed for the feature as a whole
- [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [x] **These areas should be updated or checked**: 
  1. Screenshot of the filter modal 
  2.

### CODE REVIEW

File: client/packages/common/src/intl/locales/en/common.json [hunk 1]
```diff
@@ -601,7 +601,7 @@
   "table.show-columns": "Show / hide columns",
   "title.confirm-delete-encounter": "Delete encounter",
   "warning.caps-lock": "Warning: Caps lock is on",
-  "description.notification-preferences": "By default, the system will tell you if you have more than 3 or 6 months of stock on the homepage dashboard.",
+  "description.notification-preferences": "By default, the system will tell you if you have less than 3 or 6 months of stock on the homepage dashboard.",
```
Comment (lache-melvin) [inline client/packages/common/src/intl/locales/en/common.json:604]:
```suggestion
  "description.notification-preferences": "By default, the system will tell you if you have less than 3 or more than 6 months of stock on the homepage dashboard.",
```

Is it too wordy though? Is this ugly though?

`"By default, the system will tell you if you have <3 or >6 months of stock on the homepage dashboard.`

**Review by lache-melvin — CHANGES_REQUESTED** [review]

File: client/packages/common/src/intl/locales/en/common.json (hunk 1 above)
Comment (roxy-dao) [reply client/packages/common/src/intl/locales/en/common.json]: Think it's fine to be wordy... looks weird with < 3 or > 6

**Review by lache-melvin — APPROVED** [review]
🙏
